source/docs/v3/tenancy-bootstrappers.blade.md

```diff
@@ -29,31 +29,76 @@ php artisan cache:clear --tag=tenant_123
 Note that you must use a cache store that supports tagging, e.g. Redis.
 
 ## Filesystem tenancy bootstrapper {#filesystem-tenancy-boostrapper}
+The filesystem bootstrapper makes your app's `storage_path()` and `asset()` helper and the `Storage` facade tenant-aware by modifying the paths they retrieve.
 
-This bootstrapper does the following things:
+> Note: If you want to bootstrap filesystem tenancy differently (e.g. provision an S3 bucket for each tenant), you can absolutely do that. Take a look at the package's bootstrappers to get an idea of how to write one yourself, and feel free to implement it any way you want.
+
+### Storage path helper
+
+The bootstrapper suffixes the path retrieved by `storage_path()` to make the helper tenant-aware.
+
+- The suffix is `config('tenancy.filesystem.suffix_base') . $tenantKey`
+- The suffix base is 'tenant' by default, but feel free to change it in the tenancy config
+- After suffixing the `storage_path()` helper, it returns `"/$path_to_your_application/storage/$suffix/"`
+
+Since `storage_path()` will be suffixed, your folder structure will look like this:
+
+![The folder structure](https://i.imgur.com/GAXQOnN.png)
+
+Logs will be saved in `storage/logs` regardless of any changes to `storage_path()` and regardless of the tenant.
+
+### Storage facade
+
+The bootstrapper also makes the `Storage` facade tenant-aware by suffixing the roots of disks listed in `config('tenancy.filesystem.disks')` and by overriding the roots in `config('tenancy.filesystem.root_override')`.
```

Comment:
> and by overriding the roots in `config('tenancy.filesystem.root_override')`

This part is a bit hard to understand

Reply:
How would you make it easier to understand?? I thought adding 'disk' before 'roots' might make it clearer, but I think you might mean somethings else

Reply:
Updated this a bit

Reply:
I think what I meant here is that the sentence is complex and in general the `root_override` logic isn't something people intuitively have in their mind, so it's better to explain this in a way that's clear and understandable.

Reply:
Yeah, that would be the best. I [added some info](https://github.com/stancl/tenancy-docs/pull/201/files#diff-8a5121872d845abceb00e66b26a66c06700c839e289cd160af08612cc4c71cdeR51), hope that helps to make the section understandable